eng/SnippetGenerator/SnippetGenerator.Tests/CSharpProcessorTests.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using SnippetGenerator;
+
+namespace SnippetGenerator.Tests
+{
+    public class Tests
```

Comment:
![https://media3.giphy.com/media/XaLPBWnq4Nr3qjjpq2/giphy.gif](https://media3.giphy.com/media/XaLPBWnq4Nr3qjjpq2/giphy.gif)

Reply:
Wow that's a large gif..